setup.py

```diff
@@ -53,8 +53,8 @@ def read_requirements(path):
                 processed_requirements.append(req)
         return processed_requirements
 
-
-requirements = read_requirements("requirements.txt")
+additional_requirements = ["mathgenerator @ git+https://github.com/bkb2135/mathgenerator.git@main"]
```

Comment:
let's not forget to change this when https://github.com/synapse-alpha/mathgenerator/pull/1#pullrequestreview-1855272511 is merged

Reply:
why is this necessary?